src/analyzer/protocol/stepping-stone/SteppingStone.cc

```diff
@@ -215,4 +216,12 @@ void SteppingStone_Analyzer::Done()
 	Unref(resp_endp);
 	}
 
+SteppingStoneManager* SteppingStoneManager::Get()
+	{
+	if ( ! instance && stp_correlate_pair )
+		instance = new SteppingStoneManager();
```

Comment:
Is this cleaned up anywhere?

Reply:
Yes, it gets deleted in `delete_run` in RunState.cc.